src/run.rs

```diff
@@ -266,22 +267,65 @@ pub fn clean_up(
     tmp_dir: TempDir,
     worktree_branch: Option<&str>,
     subprocesses: &mut [Child],
+    prompt: bool,
 ) -> Result<()> {
     if !delete_file.exists() {
         log::info!(
             "Project directory preserved at: {}",
             tmp_dir.into_path().display()
         );
-    } else if worktree_branch.is_some() {
-        let mut command = std::process::Command::new("git");
-        command
-            .args(["worktree", "remove"])
-            .arg(&tmp_dir.path())
-            .arg("--force");
-        ensure!(
-            command.status().context("Could not start git")?.success(),
-            "cannot remove working tree"
-        );
+    } else {
+        let delete = if prompt {
+            let delete;
+            println!("Are you sure you want to delete this project? (Y|N)");
```

Comment:
The convention in Unix and CLI is to put the default choice in uppercase and the others in lowercase.

Example:

> :: Synchronizing package databases...
> :: Starting full system upgrade...
> :: Replace wxgtk-common with extra/wxwidgets-common? [Y/n] 

`Y` is the default choice if you press enter because it is in upper case.